index.js

```diff
@@ -26,6 +26,19 @@ if (!rules) {
   );
 }
 
+function getConfig(type) {
+  return {
+    plugins: ["local-rules"],
+    rules: Object.fromEntries(rules.map((rule) => {
+      return [`local-rules/${rule}`, type]
+    }))
+  }
+}
+
 module.exports = {
+  configs: {
+    error: getConfig("error"),
```

Comment:
Let's rename `error` to `all`. It looks to be normal for `all` to set all rules to severity `error`: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/configs/all.ts